lib/ts/recipe/emailpassword/emaildelivery/index.ts

```diff
@@ -0,0 +1,110 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TypeEmailPasswordEmailDeliveryInput, User } from "../types";
+import { User as EmailVerificationUser } from "../../emailverification/types";
+import { createAndSendCustomEmail as defaultCreateAndSendCustomEmail } from "../passwordResetFunctions";
+import { NormalisedAppinfo } from "../../../types";
+import Recipe from "../recipe";
+import { normaliseAndInvokeDefaultCreateAndSendCustomEmail as normaliseAndInvokeDefaultEmailVerificationCreateAndSendCustomEmail } from "../../emailverification/emaildelivery";
+
+export async function getNormaliseAndInvokeDefaultCreateAndSendCustomEmail(
```

Comment:
This should become its own service. You can call this service `backwardCompatibilityService`